File: pset6/mario/less/mario.py
from cs50 import get_int

n = get_int('Height: ')

# Check input
while n < 1 or n > 8:
    print('Height should be 1 - 8 inclusive.')
    n = get_int('Height: ')

# Loop for half-pyramid
for i in range(1, n + 1):
    for j in range(n - i):
        print(' ', end='')
    for k in range(1, i + 1):
        print('#', end='')
        if k == i:
            print('')

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Prompt user for height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);


    //Print pyramid
    for (int j = 1; j <= n; j++)
    {
        //Print space
        for (int k = n; k > j; k--)
        {
            printf(" ");
        };

        //Print hash
        for (int i = 1; i <= j; i++)
        {
            printf("#");
        };
        printf("\n");
    }
}

